### DISCUSSION TITLE
What is the meaning of the highlighted part?

### DISCUSSION THREAD

**Discussion by hong77777** [Open-Systems-Pharmacology/Forum#1029] [Questions & Problems] [answered]
![1](https://user-images.githubusercontent.com/94173897/143771227-8e63ba15-c7dd-4ee0-b3bf-4ca924a74a29.jpg)

**Comment by AndreDlm** [comment, marked as answer]
These options denote two heuristics to estimate the capillary surface area of the organs. You can find further information in the OSP manual:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#biometrics

**Reply by hong77777** [reply]
Okay, thanks
